src/main/kotlin/com/cinephilia/payment/controllers/PaymentController.kt

```diff
@@ -0,0 +1,55 @@
+package com.cinephilia.payment.controllers
+
+import com.cinephilia.payment.commands.cancelPaymentCommand
+import com.cinephilia.payment.commands.createPaymentCommand
+import com.cinephilia.payment.commands.proceedPaymentCommand
+import com.cinephilia.payment.commands.rejectPaymentCommand
+import com.cinephilia.payment.dtos.CreatePaymentResponseDto
+import org.springframework.web.bind.annotation.*
+import ru.quipy.core.EventSourcingService
+import com.cinephilia.payment.dtos.CreatePaymentRequestDto
+import com.cinephilia.payment.dtos.ProceedPaymentDto
+import com.cinephilia.payment.dtos.RejectPaymentDto
+import com.cinephilia.payment.enitites.PaymentAggregate
+import com.cinephilia.payment.enitites.PaymentAggregateState
+import java.util.*
+
+@RestController
+@RequestMapping("/payments")
```

Comment:
This is not the generated controller. I've asked to use openapi code generation to train the API first approach